### PR TITLE
Map legend

### PR DESCRIPTION
**Pull request by chrismjohnston**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] There are no merge conflicts

### CODE REVIEW

File: fireflight/src/components/HamburgerNavigation.jsx [hunk 1]
```diff
@@ -217,8 +217,9 @@ export default function PersistentDrawerLeft({
             </ListItem>
           ))} */}
         </List>
-        <Divider />
         <MapLegend />
```
Comment (kmallen91) [inline fireflight/src/components/HamburgerNavigation.jsx]: Divider needs to be above MapLegend

Comment (chrismjohnston) [reply fireflight/src/components/HamburgerNavigation.jsx]: This has been resolved

**Comment by acruzcrespo** [comment]
@chrismjohnston Can I merge this PR?

**Comment by chrismjohnston** [comment]
Yes, please merge

**Pull request merged**